### DISCUSSION TITLE
it'd be nice to have different web shortcuts on newpage for each workspace separately. like maybe an options at least

### DISCUSSION THREAD

**Discussion by spectrum705** [zen-browser/desktop#1603] [Ideas]
### Captchas

- [X] I have read the instructions.
- [ ] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

feature request to make workspaces better.
it'd be nice to have different web shortcuts on newpage for each workspace separately. like maybe an options at least

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.2 

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1653/itd-be-nice-to-have-different-web-shortcuts-on-newpage-for-each">ZEN-1653 it'd be nice to have different web shortcuts on newpage for each workspace separately. like maybe an options at least</a></p>
